`Table` - New sorting and tooltip documentation update (#1871)
hashicorp/design-system: website/docs/components/table/partials/code/how-to-use.md

</Hds::Table>
```

### Tooltip

> Whenever possible [table headers](/table#headers) should be clear, concise, and straightforward. But there could be cases in which the label is not sufficient by itself and extra information is required. In this case it's possible to show a tooltip next to the label in the header:

Reviewer: Grammarly suggestions

```suggestion
[Table headers](/table#headers) should be clear, concise, and straightforward whenever possible. However, there could be cases where the label is insufficient by itself, and extra information is required. In this case, it's possible to show a tooltip next to the label in the header:
```

> [Table headers](/table#headers) should be clear, concise, and straightforward whenever possible. However, there could be cases where the label is insufficient by itself, and extra information is required. In this case, it's possible to show a tooltip next to the label in the header:

Reviewer 2: ```suggestion
[Table headers](/table#headers) should be clear, concise, and straightforward whenever possible. However, there could be cases where the label is insufficient by itself and extra information is required. In this case, it’s possible to show a tooltip next to the label in the header:
```

```handlebars
<Hds::Table
  @model={{this.model.myDemoData}}
  @columns={{array
    (hash key="artist" label="Artist")
    (hash key="album" label="Album" tooltip="Title of the album (in its first release)")

>     (hash key="vinyl-cost" label="Vinil Cost (USD)" isSortable=true tooltip="Cost of the vinyl (adjusted for inflation)" align="right")

Reviewer 2: ```suggestion
    (hash key="vinyl-cost" label="Vinyl Cost (USD)" isSortable=true tooltip="Cost of the vinyl (adjusted for inflation)" align="right")
```
